nextflow.config

```diff
@@ -11,28 +11,77 @@ vim: syntax=groovy
  */
 
 profiles {
-    standard {
-        includeConfig 'conf/uppmax.config'
+    cfc {
+        includeConfig 'conf/cfc.config'
```

Comment:
Does this not also need the base config?

Reply:
Fixed in a previous commit already.